src/hotspot/os/linux/cgroupV1Subsystem_linux.cpp

```diff
@@ -39,6 +39,13 @@
  * on the contents of the mountinfo and cgroup files.
  */
 void CgroupV1Controller::set_subsystem_path(char *cgroup_path) {
+  if (_cgroup_path != nullptr) {
```

Comment:
Strictly speaking the is-not-null condition is not necessary when you free stuff. nullptr are ignored on free.

Reply:
Yes. I'm aware, but wanted to be explicit here as it more clearly shows the intent. Hope that's ok.